Add draft of DSAA wrapped blog post (#31)
LKS-CHART/blog: _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd

---
title: "DSAA Wrapped"
description: |
  Reflecting on 2022.
author:
  - name: Chloe Pou-Prom, Meggie Debnath, Derek Beaton
date: 2022-12-13
output:
  distill::distill_article:
    self_contained: false
draft: true
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE)
```

> The end of the year is a time for reflection, reminiscing, and sharing screenshots of our Spotify Wrapped. For those who are unfamiliar, Spotify Wrapped is a yearly marketing campaign by Spotify, where they compile streaming data and create a personalized graphic for each user. ^[We surveyed the members on DSAA and found that the most listened music genre is: ???] 

Author: Could be fun to ask people what their top genre on Spotify Wrapped (or the Apple Music equivalent) was and we could include this here :D

Reviewer: looks like the top answer was pop (and some interesting variations of indie ~ indie soul, indie pop, indie Québécois)

In the spirit of data, end-of-year reflections, and colorful infographics, we thought it would be fun to create our very own DSAA wrapped!

![](dsaa_wrapped.PNG)

Enjoy some stats!!

# Projects

2022 was a busy year with lots of projects!!

> - Total \# of projects deployed: 7

Author: Is this number correct? I feel like there are more...
ED wait times, SMH IRT staffing optimization, chartwatch surgical, ops centre unit level tiles, hemodialysis, ICH

Reviewer 2: ID consult, CHARTWatch 2.0.

Reviewer 2: so maybe let's say at least 8-9 within the year. I do think there were some data fulfillment ones, too (e.g., a PACS one).

Reviewer 2: We have 9 big ones **from July through December**: hemo, ID consult, CW2.0, Unit Level Tiles, SMH IRT, CW Surg, Ops Centre alerting, ED wait times, ICH

- Most viewed project: Ops Centre

>     - The Ops Centre project is....

Author: Todo

Author: Also, I don't have exact view counts, but I'm assuming Ops Centre is our most used/viewed!

> - Project with the most team members whose names start with _M_^: Hemodialysis

Author: Meggie, Maitreyee, Mitch

Reviewer 2: alternate: product family with most Ms?

>     - The Hemodialysis project is...

Author: Todo

>     - The Hemodialysis project is...

Reviewer: ```suggestion
    - The Hemodialysis project is a weekly email alert that gets sent to clinicians at the St. Michael's Hospital hemodialysis clinics. The goal of the project is to predict a patient's risk of a hospital visit in the 30 days following their dialysis sessions.
```

    

# Product families

2022 was the year of _product families_.

![](https://slack-imgs.com/?c=1&o1=ro&url=https%3A%2F%2Fimages.squarespace-cdn.com%2Fcontent%2Fv1%2F58b3cae5d1758e0132a008c6%2F1582078446179-VU0VAYNPOV4P3JIBKNIY%2FIt%2527s%2BAbout%2BFamily%2BLogo%2B1500.png%3Fformat%3D1000w)

What's a product family?

> > A product family is....

Author: Do we have a definition for "product families" somewhere?

Reviewer 2: Cathy will write a blog post on it. I think she'd be happy to give us a sentence or two for this

Some product family stats!

> - Number of product families: ??

Author: I think this is... 6? Or 5?

Reviewer 2: 6, but also like 6.5. Data Fulfillment is the elusive DIG-only family, and we have special projects that don't fit into any families

> - Product family with the most projects: Patient Flow??

Reviewer: did a rough count of the sub-folders in each of the PF clickup spaces - this is what i found based on old/new projects combined!
- patient flow: 8
- ai triage: 10
- medical imaging: 2
- workforce optimization: 11
- decision guidance: 9
- data fulfillment: 3

# DSAA and improvement

2022 was a year where we focused on improving existing operations.

- \# of sprint improvement weeks: 3

    - The sprint improvement week is a week where we....
    

>     - Some notable achievements from past sprint improvement weeks are ...

Author: Todo

    
- \# of blog posts: 5
  

>     - 2022 is when we re-launched this blog! 

Author: Do we have any stats on views/traffic/engagement? We could add something here!

>     - The sprint improvement week is a week where we....

Reviewer: ```suggestion
    - The sprint improvement week is one full week dedicated to DSAA process improvement work in the areas of data infrastructure, source system optimization, DevOps, data governance, databases, analytics and project management processes, and more! The DSAA team works together to clear a backlog of tasks, and then we share our accomplishments through demos at the end of the week.
```

    
- \# of journal club sessions lead by the Advanced Analytics team: at least 16

    - We covered the following topics: [add links to ethics & DEI], anomaly detection^[blog post coming soon! :-)], survival analysis, Canadian healthcare.
    
- \# of "sheets" created: 3

- \# of Advanced Analytics mini-demos: 8

>     - Mini-demos are lead by a different Advanced Analytics team member..... Add link to Derek's funneljoin here

Author: Todo

>     - We covered the following topics: [add links to ethics & DEI], anomaly detection^[blog post coming soon! :-)], survival analysis, Canadian healthcare.

Reviewer: ```suggestion
    - We covered the following topics: [AI ethics, equity, & inclusion](https://github.com/LKS-CHART/AA_journal_club/tree/main/2022_01_Ethics_Bias_Inclusion), anomaly detection^[blog post coming soon! :-)], survival analysis, Canadian healthcare.
```

    
# DSAA's social committee

Our team's social committee (a.k.a. the Party Planning Committee) (a.k.a. the Party Planning Pandas) have organized some GREAT events this year.

- \# of steps completed during the Step Challenge: 1,983,000

    - The Step Challenge was 1-week challenge in May where team members were encouraged to get as many steps in as possible!
    
- \# of "wizard" classes: 3

    - Wizard classes are DSAA-lead classes to learn something fun. This year we learned, how to bake puff pastries, how to play euchre and how to make pizza.
    
    - Speaking of pizza....
    
- \# of pizzas made by DSAA team members: $\infty$

[add pics of pizza here???]

> - Most requested album: The Charlie Brown Christmas soundtrack
> 
>     - [something baout solo work together channel]

Reviewer: ```suggestion
- Most requested album: The Charlie Brown Christmas soundtrack

    - In an effort to maintain some of the pre-pandemic joy of working independently but with others around, we created a slack channel called #solo-work-together, where we can hop on a call and work with others while listening to some background music, which ranges from lo-fi beats to Charlie Brown!
```

    

# Product families

2022 was the year of _product families_.^[Blog post coming soon!! :fingers_crossed:]

![](https://slack-imgs.com/?c=1&o1=ro&url=https%3A%2F%2Fimages.squarespace-cdn.com%2Fcontent%2Fv1%2F58b3cae5d1758e0132a008c6%2F1582078446179-VU0VAYNPOV4P3JIBKNIY%2FIt%2527s%2BAbout%2BFamily%2BLogo%2B1500.png%3Fformat%3D1000w)

Whoa, wait. What exactly is a product family? 

- **Product** at DSAA is a _platform, application, system, or report_ that is created to meet patient, clinician, or hospital operational needs.

- **Product family** (roughly comparable to a product line) is a group of related products that are developed and maintained by a single cross-functional team at DSAA. 

- We adopted this custom framework in order to allow staff to work effectively within persistent teams and to leverage resources across products the best we can. The products under each product family are related by common product value and/or shared domain expertise requirement.

Some product family stats!

- Number of product families: 6

> - Product family with the most projects: The _Patient Flow_ product family ^[Fun fact: The _Patient Flow_ product family is also the product family with the most Fast & Furious memes.]

Author: Cathy confirmed that there are 6 families and that Patient Flow has the most projects
